src/ml/neural_net/mps_layer_conv_padding.m

```diff
@@ -0,0 +1,57 @@
+#include <ml/neural_net/mps_layer_conv_padding.h>
+
+@implementation ConvolutionPadding
+
++ (BOOL) supportsSecureCoding {
+  return true;
+}
+
+- (id) initWithParams:(int)paddingWidth
+        paddingHeight:(int)paddingHeight
+          strideWidth:(int)strideWidth
+         strideHeight:(int)strideHeight {
+  @autoreleasepool {
```

Comment:
I don't see a need for an autorelease pool here

Reply:
Typically it's a good idea to put use these in C++ functions whose implementations contained Objective C calls, but otherwise it should only be used to avoid long-lived allocations when needed

Reply:
done!